src/modal_server.py

```diff
@@ -174,6 +176,8 @@ def monitor_inactivity():
 @app.function(image=custom_image)
 @modal.wsgi_app()
 def run_server():
+    data_handler.download_data()
```

Comment:
We don't want to do this download on every server start, this should be one time (during image build) and cached in volumes.

Also, every image build after a dependency changes shouldn't refetch the data, it should use what's cached in the volume.

Reply:
changed it to one time set up script